packages/cyberstorm/src/newComponents/Icon/Icon.css

```
@@ -10,19 +10,26 @@
 
     flex-shrink: 0;
-    color: var(--icon-color, "currentColor");
+    color: var(--icon-color, currentColor);
     fill: currentcolor;
     stroke: currentcolor;
     block-size: 100%;
     inline-size: auto;
     min-inline-size: 100%;
+    max-inline-size: none;
 
     stroke-width: 0;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)